pom.xml

```diff
@@ -1,191 +1,194 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
-  <modelVersion>4.0.0</modelVersion>
-  <groupId>com.teamscale</groupId>
-  <artifactId>teamscale-upload</artifactId>
-  <packaging>jar</packaging>
-  <version>1.0-SNAPSHOT</version>
-  <name>teamscale-upload</name>
-  <organization>
-    <name>CQSE GmbH</name>
-    <url>https://www.cqse.eu/</url>
-  </organization>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
 
-  <properties>
-    <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <maven.compiler.source>1.8</maven.compiler.source>
-    <maven.compiler.target>1.8</maven.compiler.target>
-    <graalvm.version>20.3.0</graalvm.version>
-  </properties>
+    <groupId>com.teamscale</groupId>
```

Comment:
I did this in the first new commit because this touches every file and probably messes up the diff. So hopefully GitHub will let you do a diff of only the changes after this commit :-)